internal/users/users.go

```diff
@@ -34,13 +37,17 @@ type GroupInfo struct {
 type options struct {
 	groupPath  string
 	gpasswdCmd []string
+	getentCmd  []string
 }
 
 var defaultOptions = options{
 	groupPath:  "/etc/group",
 	gpasswdCmd: []string{"gpasswd"},
+	getentCmd:  []string{"getent", "passwd"},
```

Comment:
`getUsersCmd`? As we won’t pass any parameter to it, this is to have it opaque from the rest of the code.